hudi/tests/metrics.py

```diff
@@ -0,0 +1,41 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from datadog_checks.dev.jmx import JVM_E2E_METRICS_NEW
+
+METRICS = [
+    # metrics commented out can be emitted but not without adding more complex updates to the environment
```

Comment:
What are the complex updates?

Reply:
I would have to update `script.scala` to perform a clean and rollback, which could possibly not take too long, but it could also take a while to figure it out in the environment. If you think it's worth it I can try it out though!

Reply:
I think it should be fine without it, but I recommend just writing down what to do if someone does need to test those metrics later down the line.